exporter/cassandraexporter/config.go

```diff
@@ -6,6 +6,7 @@ package cassandraexporter // import "github.com/open-telemetry/opentelemetry-col
 type Config struct {
 	DSN         string      `mapstructure:"dsn"`
 	Port        int         `mapstructure:"port"`
+	Timeout     int         `mapstructure:"timeout"`
```

Comment:
Any reason you're using `int` instead of `time.Duration`? Most exporters are using `time.Duration` directly, so it would make sense to be consistent unless there are specific reasons not to.